opencti-platform/opencti-front/src/utils/tests/overviewLayoutCustomization.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { describe, it, expect } from 'vitest';
+import { renderHook } from '@testing-library/react';
+import React from 'react';
+import useAuth from '../hooks/useAuth';
+import { createMockUserContext, ProvidersWrapper, ProvidersWrapperProps } from './test-render';
+
+describe('overviewLayoutCustomization', () => {
+  const threatActorIndividualEntityType = 'Threat-Actor-Individual';
+  const overviewLayoutCustomization = new Map();
+  const threatActorIndividualConfiguration = new Map([
+    ['details', 6],
+    ['basicInformation', 6],
+    ['demographics-biographics', 6],
+    ['latestCreatedRelationships', 6],
+    ['latestContainers', 6],
+    ['externalReferences', 6],
+    ['mostRecentHistory', 6],
+    ['notes', 12],
+  ]);
+  overviewLayoutCustomization.set(
```

Comment:
same here, you can directly construct your map

Reply:
I can see why it was better to refactor the multiple set instructions.
For this last one, I do not understand. Why should this be constructed differently?

Reply:
It avoids the creation of an empty map some lines above